test/application/modules/legacy/commands/register_keys.spec.ts

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseCommand, cryptography, VerifyStatus } from 'lisk-sdk';
+
+import {
+	COMMAND_ID_REGISTER_KEYS,
+	COMMAND_NAME_REGISTER_KEYS,
+} from '../../../../../src/application/modules/legacy/constants';
+
+import { RegisterKeysCommand } from '../../../../../src/application/modules/legacy/commands/register_keys';
+import { registerKeysParamsSchema } from '../../../../../src/application/modules/legacy/schemas';
+
+const { getRandomBytes } = cryptography;
+
+const getContext = (params, publicKey, getAPIContext): any => {
+	const senderPublicKey = Buffer.from(publicKey, 'hex');
+	return {
+		params,
+		transaction: {
+			senderPublicKey,
+		},
+		getAPIContext,
+	} as any;
+};
+
+describe('Register BLS Keys command', () => {
+	let registerBLSKeyCommand: RegisterKeysCommand;
+
+	const getStore: any = () => ({
+		getWithSchema: jest.fn(),
+		has: jest.fn(),
+		del: jest.fn(),
+	});
+
+	const getAPIContext: any = () => ({
+		getStore,
+	});
+
+	beforeEach(() => {
+		registerBLSKeyCommand = new RegisterKeysCommand(COMMAND_ID_REGISTER_KEYS);
+	});
+
+	it('should inherit from BaseCommand', () => {
+		expect(RegisterKeysCommand.prototype).toBeInstanceOf(BaseCommand);
+	});
+
+	describe('constructor', () => {
+		it('should have valid id', () => {
+			expect(registerBLSKeyCommand.id).toBe(COMMAND_ID_REGISTER_KEYS);
+		});
+
+		it('should have valid name', () => {
+			expect(registerBLSKeyCommand.name).toBe(COMMAND_NAME_REGISTER_KEYS);
+		});
+
+		it('should have valid schema', () => {
+			expect(registerBLSKeyCommand.schema).toEqual(registerKeysParamsSchema);
+		});
+	});
+
+	describe('verify', () => {
+		const publicKey = 'ac8fb4c7318a1ff9e399102f4b87e3d831e734a48013967bfdba978c9313455c';
+		const transactionParams = {
+			blsKey: getRandomBytes(48),
+			proofOfPossession: getRandomBytes(96),
+			generatorKey: getRandomBytes(32),
+		};
+
+		it(`should return status Ok`, async () => {
+			const commandVerifyContextInput = getContext(transactionParams, publicKey, getAPIContext);
+			const getValidatorAccount = jest.fn().mockReturnValue({ generatorKey: getRandomBytes(32) });
+			registerBLSKeyCommand.addDependencies({ getValidatorAccount } as any);
+			await expect(registerBLSKeyCommand.verify(commandVerifyContextInput)).resolves.toHaveProperty(
+				'status',
+				VerifyStatus.OK,
+			);
```

Comment:
Please consider restructuring the it to following. Please remove `await` if it is unnecessary.
```suggestion
		expect(registerBLSKeyCommand.verify(commandVerifyContextInput))
			.resolves.toHaveProperty('status', VerifyStatus.OK);
```

Reply:
Restructuring is not working because `npm run format`  updates the code back to:
```
	await expect(registerBLSKeyCommand.verify(commandVerifyContextInput)).resolves.toHaveProperty(
		     'status',
		     VerifyStatus.OK,
	 ); 
```
Also, removing await leads to lint fail.